app/models/youth_risk_assessment.rb

```diff
@@ -17,6 +17,6 @@ def previous_assessment
   end
 
   def move_from_location
-    errors.add(:move, "'from_location' must be from either a secure training centre or a secure children's home") unless move&.from_location&.secure_training_centre? || move&.from_location&.secure_childrens_home?
+    errors.add(:move, "'from_location' must be from either a secure training centre or a secure children's home") unless move&.from_location&.secure_training_centre? || move&.from_location&.secure_childrens_home? || move&.from_location&.young_offender_institution
```

Comment:
It might be a nice option in future to move this logic to the `Location` model but this is absolutely fine now. Perhaps if the complexity grows any further....